src/logDownloaderService.py

```diff
@@ -0,0 +1,79 @@
+import os
+import time
+import hashlib
+import subprocess
+import psutil
+from git import Repo
+import shutil
+from loguru import logger
```

Comment:
It's great that you log! However:
1. It's better to log into a file, I'm not sure whether you can see the output in the Task Scheduler.
2. The logging here is very simple, you can use the built-in `logging` module.